### PR TITLE
Different BPN for each participant

### PR DESCRIPTION
**Pull request by hemantxpatel**
## Description
Different BPN for each participant (alice / bob / miw)
* Assign separate BPNs
* Create wallets for each participants into miw.
* Update keycloak config

Closes https://github.com/eclipse-tractusx/tutorial-resources/issues/26
## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by wolf4ood** [comment]
@hemantxpatel 

which was the problem that you were having ? From MIW configuration prospective the PR looks good. 
I had problem when doing the contract negotiation. From what i saw each participant alice and bob are using the same client_id `miw_private_client` which is not correct as they will receive the same VC/VP from the MIW in this way.

Alice and Bob should have their own client credentials which is different from the authority one which is `miw_private_client`

**Comment by paullatzelsperger** [comment]
So far I have seen the following problems:
- as @wolf4ood said, the Keycloak config is not complete. You'll need to create clients (with separate client-ids and client-secrets), roles and service accounts, etc.
- The `miw-seed.json` collection explicitly seeds wallet 1 and 2, but that should be handled through env vars. will make future improvements easier (--> "trudy")
- the requests in `miw-seed.json` should not be in an own collection, but should be in a subfolder in the `mxd-seed.json` collection

The following [gist]() is a quick attempt at a keycloak config, take a look.

**Comment by hemantxpatel** [comment]
Now, each participants (alice, bob, miw) have their own client in keycloak.

File: mxd/keycloak.tf [hunk 1]
```diff
@@ -94,8 +94,13 @@ resource "kubernetes_config_map" "keycloak_env" {
     KEYCLOAK_ADMIN             = "admin"
     KEYCLOAK_ADMIN_PASSWORD    = "admin"
     # the KC_HOSTNAME must be known in advance, so that Keycloak's token contain valid `iss` claims
-    KC_HOSTNAME       = local.keycloak-ip
+
+    KC_HOSTNAME = local.keycloak-ip
+    #    KC_HOSTNAME_URL   = "http://localhost:8080"
```
Comment (wolf4ood) [inline mxd/keycloak.tf:99]: I would remove this

**Review by wolf4ood — APPROVED** [review]
LGTM just a minor nit. Tried in my local env and catalog/negotiation/transfer worked file

File: mxd/keycloak.tf (hunk 1 above)
Comment (hemantxpatel) [reply mxd/keycloak.tf:99]: Will do that. That `KC_HOSTNAME_URL` instead of `KC_HOSTNAME` is  needed to work Keycloak UI.

Comment (hemantxpatel) [reply mxd/keycloak.tf]: Done

**Pull request merged**